### PR TITLE
Add global flags per applications.

### PR DESCRIPTION
**Pull request by jaleszek**
Admin/Assessor can set up the global flag on the comments form. It should be taken into consideration in flags number indicator on the applications list. Admin importance flag is set up by Admin on the Admin comments section, the Assessor importance flag is set up by Assessor on Critical comments.

### CODE REVIEW

File: app/controllers/concerns/form_answer_flags_mixin.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+module FormAnswerFlagsMixin
+  def toggle
```
Comment (matthewford) [inline app/controllers/concerns/form_answer_flags_mixin.rb]: Assignment Branch Condition size for toggle is too high. [24.08/15]

File: config/routes.rb [hunk 2]
```diff
@@ -98,6 +98,10 @@
         end
       end
       member { get(:review) }
+      resources :flags, only: [] do
+        collection{ post :toggle }
+      end
+
```
Comment (matthewford) [inline config/routes.rb]: Extra empty line detected at block body end.

File: config/routes.rb [hunk 3]
```diff
@@ -98,6 +98,10 @@
         end
       end
       member { get(:review) }
+      resources :flags, only: [] do
+        collection{ post :toggle }
```
Comment (matthewford) [inline config/routes.rb]: Space missing to the left of {.

File: config/routes.rb [hunk 4]
```diff
@@ -121,7 +125,7 @@
       end
 
       resources :flags, only: [] do
-        collection{ get :toggle }
+        collection{ post :toggle }
```
Comment (matthewford) [inline config/routes.rb]: Space missing to the left of {.

**Comment by RuslanHamidullin** [comment]
Looks good @jaleszek :beers:

**Pull request merged**